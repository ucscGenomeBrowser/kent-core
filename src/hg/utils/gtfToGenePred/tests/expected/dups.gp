MGC2.1.1.1.1.A11.1.fa	chr16	-	11495971	11502185	11495971	11502185	2	11495971,11502160,	11496117,11502185,
mgc9_chr1.111.008.a	chr1	+	110421593	110453108	110421593	110453108	11	110421593,110426723,110428636,110429442,110431292,110446635,110447169,110449242,110450249,110452100,110452739,	110421879,110426881,110428814,110429624,110431403,110446877,110447362,110449435,110450409,110452263,110453108,
