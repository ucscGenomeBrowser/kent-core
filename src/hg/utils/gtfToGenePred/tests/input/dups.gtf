chr4	mgc3_chr4.108.108.seq.masked.005.a	CDS	107637159	107637204	.	-	.	gene_id "MGC3.4.1.2.1.C10.1.fa"; transcript_id "MGC3.4.1.2.1.C10.1.fa";
chr4	mgc3_chr4.108.108.seq.masked.005.a	CDS	107645804	107646069	.	-	.	gene_id "MGC3.4.1.2.1.C10.1.fa"; transcript_id "MGC3.4.1.2.1.C10.1.fa";
chr4	mgc3_chr4.108.108.seq.masked.005.a	CDS	107637159	107637204	.	+	.	gene_id "MGC3.4.1.2.1.C10.1.fa"; transcript_id "MGC3.4.1.2.1.C10.1.fa";
chr4	mgc3_chr4.108.108.seq.masked.005.a	CDS	107645804	107646069	.	+	.	gene_id "MGC3.4.1.2.1.C10.1.fa"; transcript_id "MGC3.4.1.2.1.C10.1.fa";
chr16	mgc2_iscan_int_chr16.12.007.a	CDS	11495972	11496117	.	-	.	gene_id "MGC2.1.1.1.1.A11.1.fa"; transcript_id "MGC2.1.1.1.1.A11.1.fa";
chr16	mgc2_iscan_int_chr16.12.007.a	CDS	11502161	11502185	.	-	.	gene_id "MGC2.1.1.1.1.A11.1.fa"; transcript_id "MGC2.1.1.1.1.A11.1.fa";
chr1	iscan	start_codon	110421594	110421596	0	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110421594	110421879	763	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110426724	110426881	792	+	2	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110428637	110428814	242	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110429443	110429624	650	+	2	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110431293	110431403	496	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110446636	110446877	1004	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110447170	110447362	760	+	1	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110449243	110449435	987	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110450250	110450409	553	+	2	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110452101	110452263	697	+	1	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	CDS	110452740	110453105	986	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
chr1	iscan	stop_codon	110453106	110453108	0	+	0	gene_id "chr1.111.008"; transcript_id "mgc9_chr1.111.008.a";
