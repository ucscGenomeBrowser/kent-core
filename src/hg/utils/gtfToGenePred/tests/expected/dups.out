invalid gffGroup detected on line: chr4	mgc3_chr4.108.108.seq.masked.005.a	CDS	107637159	107637204	0.000000	-	.	gene_id "MGC3.4.1.2.1.C10.1.fa"; transcript_id "MGC3.4.1.2.1.C10.1.fa"; 
GFF/GTF group MGC3.4.1.2.1.C10.1.fa on chr4+, this line is on chr4-, all group members must be on same seq and strand
1 errors
